src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementCheckbox.tsx

```diff
@@ -0,0 +1,152 @@
+  },
+  roleDescription2: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.PermissionManagementDialog.PermissionManagementCheckbox.roleDescription2',
+    defaultMessage: 'Coming soon...',
+  },
+  roleDescription3: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.PermissionManagementDialog.PermissionManagementCheckbox.roleDescription3',
+    defaultMessage: 'Set permissions in the active domain, and any subdomain.',
+  },
+  // We don't have architecture_subdomain (which would be 4)
+  roleDescription5: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.PermissionManagementDialog.PermissionManagementCheckbox.roleDescription5',
+    defaultMessage: 'Fund expenditures and transfer funds between domains.',
+  },
+  roleDescription6: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.PermissionManagementDialog.PermissionManagementCheckbox.roleDescription6',
+    defaultMessage: 'Create and manage expenditures.',
+  },
+  tooltipNoPermissionsText: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.PermissionManagementDialog.PermissionManagementCheckbox.tooltipNoPermissionsText',
+    defaultMessage: 'You do not have permission to set the {roleName} role.',
+  },
+  tooltipNoRootDomainSelected: {
+    id:
+      // eslint-disable-next-line max-len
+      'dashboard.PermissionManagementDialog.PermissionManagementCheckbox.tooltipNoRootDomainSelected',
+    defaultMessage: 'Switch domain to #Root to set the root role.',
+  },
+});
+
+interface Props {
+  asterisk: boolean;
+  disabled: boolean;
+  role: ColonyRole;
+  domainId: number;
+}
+
+const displayName =
+  'dashboard.PermissionManagementDialog.PermissionManagementCheckbox';
+
+const PermissionManagementCheckbox = ({
+  asterisk,
+  disabled,
+  role,
+  domainId,
+}: Props) => {
+  const roleNameMessage = { id: `role.${role}` };
+  const roleDescriptionMessage = useMemo(
+    () =>
+      MSG[`roleDescription${role}`] || {
+        id: '',
+        defaultMessage: '',
+      },
+    [role],
+  );
+
+  const { formatMessage } = useIntl();
+
+  const formattedRole = formatMessage(roleNameMessage);
+
+  const checkboxContent = useMemo(
+    () => (
+      <Checkbox
+        className={styles.permissionChoice}
+        value={role}
+        name="roles"
+        disabled={disabled}
+      >
+        <span className={styles.permissionChoiceDescription}>
+          <PermissionsLabel
+            permission={role}
+            name={formattedRole}
+            inherited={asterisk}
+          />
+          <FormattedMessage {...roleDescriptionMessage} />
+        </span>
+      </Checkbox>
+    ),
+    [asterisk, disabled, role, roleDescriptionMessage, formattedRole],
+  );
+
+  const tooltipText =
+    domainId !== ROOT_DOMAIN_ID && formattedRole === 'Root'
+      ? MSG.tooltipNoRootDomainSelected
+      : MSG.tooltipNoPermissionsText;
+
+  return disabled ? (
+    <Popover
```

Comment:
Please use the actual `Tooltip` component from core, not the `Popover`.

Eg: `import { Tooltip } from '~core/Popover';`